packages/aws-cdk-lib/aws-elasticloadbalancingv2/README.md

```diff
@@ -568,7 +568,7 @@ const lb = new elbv2.NetworkLoadBalancer(this, 'LB', {
 });
 ```
 
-**Note**: The `minimumCapacityUnit` value is evenly distributed across all active Availability Zones (AZs) for the network load balancer. The distributed value per AZ must be between 2,750 and 45,000 units.
+**Note**: The `minimumCapacityUnit` value is evenly distributed across all active Availability Zones (AZs) for the network load balancer. The distributed value per AZ must be between 2,750 and 90,000 units.
```

Comment:
why 90,000 do we have the link to the doc?

Reply:
can we merge this PR branch to https://github.com/pahud/aws-cdk/tree/fix-34582 ?